x-pack/plugins/ml/public/services/results_service.js

```diff
@@ -1398,7 +1398,7 @@ function getEventRateData(
 // Extra query object can be supplied, or pass null if no additional query.
 // Returned response contains a results property, which is an object
 // of document counts against time (epoch millis).
-const SAMPLER_TOP_TERMS_SHARD_SIZE = 200;
+const SAMPLER_TOP_TERMS_SHARD_SIZE = 50000;
```

Comment:
The data visualizer uses a default `SAMPLER_TOP_TERMS_SHARD_SIZE` of 5000. I wonder if we're safe to use a 10x higher value here, bearing in mind that we're only doing one query?